src/js.rs

```diff
@@ -374,6 +374,12 @@ extern "C" {
 extern {
     pub type Generator;
 
+    /// The return() method returns the given value and finishes the generator.
+    /// 
+    /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Generator/return
+    #[wasm_bindgen(method, structural, js_name = return)]
+    pub fn gen_return(this: &Generator, value: &JsValue) -> JsValue;
```

Comment:
In the WebIDL frontend, we mangle Rust keywords into identifiers by appending an underscore, so I think we should be consistent and do the same here:

```rust
#[wasm_bindgen(method, structural, js_name = return)] 
pub fn return_(this: &Generator, value: &JsValue) -> JsValue;
```